### ISSUE TITLE
Symlink for multiple server permissions

### ISSUE THREAD

**Issue by Seeroh** [lucko/LuckPerms#460]
I know I am basically being stubborn here, but I prefer to have easy text access for fast editing for my permissions, but I would like to expand to using bungee, and have all the different permissions connected, while not having to switch to MySQL.

I can create a symlink for the groups/users/tracks folders, and still have individual configs for each server, but I am just wondering if this would be liable to cause any problems? Each time the files are edited, they should automatically update for each server, but is there any kind of file lock that ever occurs?

Basically, just wondering if this is a viable workaround, before I spend a few hours getting it all tweaked, to find that it doesn't work.

**Comment by lucko** [comment]
It should work fine, however, I'm not sure if the [`watch-files`](https://github.com/lucko/LuckPerms/wiki/Configuration#watch-files) feature will work.

As for file locking, LuckPerms doesn't do anything special to lock the storage files. Depends on your filesystem I guess? It shouldn't be a problem on unix systems.

I'd just give it a try, and let me know if you experience any problems.

**Comment by Seeroh** [comment]
I'll report back on if the watch files feature works.

**Comment by Seeroh** [comment]
It does indeed activate the filewatcher feature, so it seems to work seamlessly!

**Comment by lucko** [comment]
Yay

**Comment by FrozenBeard** [comment]
Exact same thing I want to do! I dont want to use MySQL so I can easily see/edit my permissions, stoked that this should work. will be migrating to this.
